constants/chatbotCommands.py

```diff
@@ -2793,3 +2793,32 @@ async def disable_profiling(fro: str, chan: str, message: list[str]) -> str:
     profiling.profiler.disable()
     profiling.profiler.dump_stats("stats.dump")  # TODO: persist in s3
     return "Profiler disabled. Dumped stats to disk @ stats.dump"
+
+
+@command(trigger="!leaderboardsize", syntax="<size>", hidden=True)
+async def leaderboard_size_cmd(fro: str, chan: str, message: list[str]) -> str:
+    userID = await user_utils.get_id_from_username(
+        fro,
+    )
+    user_privs = await user_utils.get_privileges(userID)
+
+    if not user_privs & privileges.USER_PREMIUM:
+        return (
+            "The leaderboardsize command is only available to Akatsuki premium users."
+        )
```

Comment:
this should probably be removed in favour of passing `privs` to the `command` decorator. for example, doing that would remove `!leaderboardsize` as a command on `!help` for non-premium users.

Reply:
resolved in 6b2ba05cd3599c8d360bec4ab35bb868b0362624